pkg/services/cosmosdb/catalog.go

```diff
@@ -9,16 +9,16 @@ func (m *module) GetCatalog() (service.Catalog, error) {
 			service.NewService(
 				&service.ServiceProperties{
 					ID:   "6330de6f-a561-43ea-a15e-b99f44d183e6",
-					Name: "azure-cosmos-document-db",
-					Description: "Azure DocumentDB (Experimental) provided by CosmosDB " +
-						"and accessible via SQL (DocumentDB), Gremlin (Graph), and Table " +
-						"(Key-Value) APIs",
+					Name: "azure-cosmos-db-sql",
+					Description: "Azure Cosmos DB is a globally distributed, " +
```

Comment:
This description is inconsistent with others. Every existing one I've looked at is just a noun (even if it's long like "Blah blah blah provided by blah blah"), but none of the others are complete sentences describing the service. That which is listed as the description here might be better suited as the `LongDescription` (under `Metadata`), but even then, it seems we haven't used complete sentences.

I know this is super nitpicky. If we can just stay consistent with these existing conventions for now, we can always circle back later and update descriptions and long descriptions across the board to comport with new conventions if we decide they need to change.